doc/sphinx/advanced_methods.rst

```diff
@@ -397,711 +397,1003 @@ ibm\_triel, ibm\_tribend and ibm\_volCons:
-same ``part-type``. One can have more objects with the same type of
-particles, but this is not recommended, because the interactions between
-objects are set up using these types.
+::
 
-angles in radians, by which the object is initially rotated around the
-:math:`x, y, z` axis. Default values are 0.0 0.0 0.0.
+    cell = OifCell(cellType=cell_type, partType=0, origin=[5.0,5.0,3.0])
 
-this parameter refers to the mass of one particle (one mesh point of the
-triangulation). For the proper setting, the mass of the whole membrane
-must be distributed to all mesh points. Default value is 1.0.
+Next, an actual object is created and its initial position saved to a
+*.vtk* file (the directory ``output/sim1`` needs to exist before the
+script is run). Each object has to have a unique ID, specified using the
+keyword ``partType``. The IDs have to start at 0 and increase
+consecutively. The other two mandatory arguments are ``cellType`` and
+``origin``. ``cellType`` specifies which previously defined cell type
+will be used for this object. ``origin`` gives placement of object’s
+center in the simulation box.
 
-.. _Mesh analysis:
 
-Mesh analysis
-^^^^^^^^^^^^^
 
-oif\_mesh\_analyze nodes-file triangles-file
+Specification of fluid and movement
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+::
 
-This command is useful for some preparatory work with mesh before it is
-used for creating elastic objects.
+    lbf = espressomd.lb.LBFluid(agrid = 1, dens = 1.0, visc = 1.5, 
+        tau = time_step, fric = 1.5, ext_force = [0.002, 0.0, 0.0])
+    system.actors.add(lbf)
 
-- file with coordinates of the mesh nodes. The center of the object
-  should be as close to (0,0,0) as possible.
+This part of the script specifies the fluid that will get the system
+moving. Here ``agrid`` :math:`=\Delta x` is the spatial discretisation
+step, ``tau`` is the time step that will be the same as the time step
+for particles, viscosity ``visc`` and density ``dens`` of the fluid are
+physical parameters scaled to lattice units. ``fric`` is a
+(non-physical) friction parameter that enters the fluid-object
+interaction and has to be set carefully. Finally, ``ext_force`` sets the
+force-per-unit-volume vector that drives the fluid. Another option to
+add momentum to fluid is by specifying the velocity on the boundaries.
 
-- file with incidences for all triangles. Each line of this file
-  contains three integer IDs (starting from 0) with indices of three
-  vertices forming one triangle.
 
-checks whether all triangles of the surface mesh are properly oriented.
-For now, only works for convex (or almost convex) objects.
+Here we achieved the movement of the fluid by applying external force.
+It can also be done by setting the velocity of the individual
+``lbnodes`` on one side of the channel and letting the flow develop, but
+this is not a very convenient setup because it has to be done at every
+integration step and the tcl-C communication slows down the computation.
```

Comment:
tcl?

Reply:
Comment deleted, because moving the fluid with LB nodes has almost never been used in OIF